account_fiscal_year/models/date_range.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Author: Damien Crier
+# Copyright 2016 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import models
+from openerp.tools import DEFAULT_SERVER_DATE_FORMAT
+
+
+class DateRange(models.Model):
+    _inherit = 'date.range'
+
+    def find_daterange_fy_start(self, date):
```

Comment:
Oh ok. I see what you meant. I remove it and update the code in the module where this method was called.